### DISCUSSION TITLE
建议添加一些快捷键

### DISCUSSION THREAD

**Discussion by Ohcecho** [oflg/fishing#40] [Ideas]
比如说打开新条目，或者那种输入副标题和条目内容的快捷键，这样就不需要操作鼠标，在制作条目的时候会更加方便。

**Comment by oflg** [comment]
确实快捷键更方便，不过这些快捷键tiddlywiki已经自带了
新条目：alt+n
切换标题、副标题、内容：（shift+）Tab

**Reply by Ohcecho** [reply]
感谢，我之前一直找不到这些快捷键
